src/core/core.layouts.js

```diff
@@ -122,8 +122,8 @@ module.exports = {
 	/**
 	 * Sets (or updates) options on the given `item`.
 	 * @param {Chart} chart - the chart in which the item lives (or will be added to)
-	 * @param {Object} item - the item to configure with the given options
-	 * @param {Object} options - the new item options.
+	 * @param {object} item - the item to configure with the given options
```

Comment:
Is this an `ILayoutItem`?

Reply:
I don't think so. Here's where it's called:
https://github.com/chartjs/Chart.js/blob/master/src/plugins/plugin.legend.js#L524
https://github.com/chartjs/Chart.js/blob/master/src/plugins/plugin.title.js#L201

Reply:
Legend and title are both layout items

Reply:
done